### PR TITLE
add a video of geometry edition

### PR DESCRIPTION
**Pull request by lucienicolier**

### CODE REVIEW

File: en/fieldwork/digitize.rst [hunk 1]
```diff
@@ -64,24 +64,25 @@ To edit, tap on **Menu button -> pencil.**
...
 It is possible to move and delete vertices of geometries and add new ones.
 
-.. container:: clearer text-left
+demonstration
+-------------
 
-  .. figure:: ../images/edit_geom_feature.gif 
-     :width: 300px
-     :alt: edit_geom_feature
+A video demonstration of some edition possibilities:
```
Comment (m-kuhn) [inline en/fieldwork/digitize.rst:80]: Is this for splitting, vertex editing or all of it?

Comment (m-kuhn) [reply en/fieldwork/digitize.rst:80]: @lucienicolier did you see this question=

Comment (lucienicolier) [reply en/fieldwork/digitize.rst:80]: @m-kuhn  no I didn't see. There is vertex, spilt, and ring on this video

File: en/fieldwork/digitize.rst [hunk 2]
```diff
@@ -64,24 +64,38 @@ To edit, tap on **Menu button -> pencil.**
...
+
+It is possible to split an existing geometry by drawing a line where it should be split.
+
 Vertex tool
 -----------
 
 It is possible to move and delete vertices of geometries and add new ones.
 
-.. container:: clearer text-left
+demonstration of geometries edition
```
Comment (m-kuhn) [inline en/fieldwork/digitize.rst:77]:
```suggestion
Demonstration of geometries edition
```

**C**apitalize titles

File: en/fieldwork/digitize.rst (hunk 1 above)
Comment (m-kuhn) [reply en/fieldwork/digitize.rst:80]: Thanks, nice !

I think "editing" is a more appropriate term than edition.

File: en/fieldwork/digitize.rst [hunk 3]
```diff
@@ -64,24 +64,38 @@ To edit, tap on **Menu button -> pencil.**
...
 
-It is possible to split an existing geometry by drawing a line where it should be split.
+  .. figure:: ../images/edit_geom.webp
+     :width: 600px
+     :alt: edit geom
+
+Freehand digitizing
+-------------------
+
+Freehand digitizing mode allowing users to 'draw' lines and polygons using their stylus/pen or mouse.
```
Comment (m-kuhn) [inline en/fieldwork/digitize.rst:91]:
```suggestion
The freehand digitizing mode allows users to 'draw' lines and polygons using their stylus/pen or mouse.
```

File: en/fieldwork/digitize.rst (hunk 1 above)
Comment (lucienicolier) [reply en/fieldwork/digitize.rst]: done

**Comment by lucienicolier** [comment]
@m-kuhn  I think I added all editing subjects or do you see something missing?

File: en/fieldwork/digitize.rst [hunk 4]
```diff
@@ -99,3 +99,15 @@ to editing mode and a line or polygon vector is selected.
   .. figure:: ../images/freehand_drawing.webp
      :width: 600px
      :alt: freehand drawing
+
+Topology
+--------
+
+If the topology is activated on the project QGIS before exporting for QField, shared nodes are treated
+as such when moving / deleting vertices, and additional vertices are added on nodes overlapping segments.
```
Comment (m-kuhn) [inline en/fieldwork/digitize.rst:107]: @nirvn I think you wrote this? "nodes overlapping segments" sounds strange to me. Maybe better "nodes placed on segments"?

Comment (nirvn) [reply en/fieldwork/digitize.rst:107]: Either way reads fine to me, if you prefer placed on segments, +1 :)

**Comment by lucienicolier** [comment]
@m-kuhn I think it is missing multiple editing, is it the right pace in your opinion?

**Comment by nirvn** [comment]
Videos are nice. Quick comment: it'd be good if we could get rid of the software watermark in the video.

**Comment by lucienicolier** [comment]
@nirvn  Do you mean reframing?

**Comment by nirvn** [comment]
@lucienicolier , this:
image

It's at least present on the freehand drawing video.

**Comment by m-kuhn** [comment]
I think that's "mine". I can redo it, what software would you recommend?

**Comment by nirvn** [comment]
@m-kuhn , I like https://github.com/Genymobile/scrcpy , I find it to be the cleanest way to capture video for documentation material.

**Comment by lucienicolier** [comment]
@m-kuhn I change the videos by vimeo videos, do you think it's better?

**Review by mbernasocchi — APPROVED** [review]

**Comment by lucienicolier** [comment]
@m-kuhn did you remake the video or could I ask @LindaOPENGIS ? Since she has a tablet?

**Comment by m-kuhn** [comment]
Yes, that would be good, I don't have a tablet with stylus at the moment.

**Comment by lucienicolier** [comment]
Hi @LindaOPENGIS, can you make a video with the freehand edition? Thanks

**Comment by LindaOPENGIS** [comment]
Ciao Lucie

cette semaine je peux pas travailler pour opengis parce que j'ai un course
formation d'avalanches e randonée. mais peut etre j'ai déjà fait ce que tu
veux ?-> regarde sur notres videos en vimeo:  https://vimeo.com/516616288 .
j'ai fait là le video pour la documentation de QField
https://docs.google.com/document/d/1tG-OH9r9vV_eZvFvLHCg5tE3aRFxK22l/edit
(seulement allemand pour l'instant). c'est ca qu'il te faut? si non je peux
m'occuper l'autre semaine.







Lucie Nicolier ***@***.***> schrieb am Mo., 22. März 2021,
09:09:

**Review by lindacamathias — APPROVED** [review]

**Pull request merged**